recipes-core/aziot-keys/aziot-keys_1.5.0.bb

```diff
@@ -519,6 +517,10 @@ SUMMARY = "aziot-keys is the default implementation of cryptographic operations
 HOMEPAGE = "https://azure.github.io/iot-identity-service/"
 LICENSE = "MIT"
 
+do_patch() {
+    sed -i '/panic = "abort"/d' ${WORKDIR}/git/Cargo.toml
```

Comment:
is having `abort` strategy better than `unwind`

Reply:
It's best to have it unspecified, so that RUST_PANIC_STRATEGY applies here. The libstd-rs runtime needs to have support built-in for the strategy or you will get compile time errors.

This deletes it from the Cargo.toml file.